src/cockpit/beiboot.py

```diff
@@ -210,6 +211,8 @@ class SshPeer(Peer):
     def __init__(self, router: Router, destination: str, args: argparse.Namespace):
         self.destination = destination
         self.always = args.always
+        self.tmpdir = tempfile.TemporaryDirectory()
+        self.known_hosts_file = Path(self.tmpdir.name) / 'user-known-hosts'
```

Comment:
In the bastion host case it is the initial thing launched by cockpit-ws (instead of cockpit-ssh). Keeping the known hosts file around throughout its lifetime seems fine to me?

For the host switcher it doesn't run, as remote.py uses its own `SshPeer` implemenation -- it doesn't even import beiboot.py (this is meant to be a top-level executable).

Reply:
If the bastion host has multiple users, do they use the same `beiboot` instance? And if not that would require some locking?

Reply:
No, every user *session* (even for the same user) gets its own cockpit-beiboot instance. It's the top-level session program (similar to `cockpit-session`).

Reply:
Ok, then this is resolved.